styling.py

```diff
@@ -0,0 +1,39 @@
+import streamlit as st
+
+# Load custom CSS
+def load_css(file_name):
+    with open(file_name) as f:
+        st.markdown(f"<style>{f.read()}</style>", unsafe_allow_html=True)
+        
+# Function to render the two-column layout with project description and calculators
+def render_layout():
+    st.title("Predictive Calc 🧮")
+    st.write("Welcome to the **Predictive Calc** app! Your go-to tool for making smarter financial decisions.")
+    
+    # Two-column layout
+    col1, col2 = st.columns(2)
+
+    with col1:
+        # Project description section
+        st.subheader("Project Description")
+        st.write(
+            """
+            This application offers a set of calculators to assist with various financial decisions, including:
+            - House Price Predictions
+            - Loan Eligibility Assessments
+            - Stress Level Detection based on social media interactions
```

Comment:
1. Sync your branch to ensure it's up to date with the latest changes, because latest changes aren't visible rn. Or maybe you resolved merge conflicts incorrectly.
2. Add links to the model names referring to their pages url.
Eg: `[House Price Predictions](URL)`